testkit/jvm/src/main/scala/zio/testkit/TestClock.scala

```diff
@@ -17,17 +17,22 @@
 package zio.testkit
 
 import java.util.concurrent.TimeUnit
+import java.time.ZoneId
 
 import zio._
 import zio.duration.Duration
 import zio.clock.Clock
 import zio.testkit.TestClock.Data
+import java.time.{ Instant, OffsetDateTime }
 
 case class TestClock(ref: Ref[TestClock.Data]) extends Clock.Service[Any] {
 
   final def currentTime(unit: TimeUnit): UIO[Long] =
     ref.get.map(data => unit.convert(data.currentTimeMillis, TimeUnit.MILLISECONDS))
 
+  final def currentDateTime: UIO[OffsetDateTime] =
```

Comment:
Can you add one test for this?

Reply:
Hey you’re welcome! So it doesn’t come up here because of the sequence of commit/ reset - force push I did, but there is actually not 1 but 2! additional tests in ClockSpec - if you look in the overall changes you should see them. Let me know if you feel a test is missing though. 👍